TodoList.Application/CurrentUserProvider.cs

```diff
@@ -1,11 +1,43 @@
-﻿namespace TodoList.Application
+﻿using Microsoft.AspNetCore.Http;
+using System.Security.Claims;
+using TodoList.Application;
+
+public class CurrentUserProvider : ICurrentUser
```

Comment:
Application layer should not depend on ASP framework details (Microsoft.AspNet.Identity.EntityFramework)

TODO: Move to Persistent